raiden/tests/integration/long_running/test_settlement.py

```diff
@@ -567,7 +566,7 @@ def test_channel_withdraw_expired(
         total_withdraw=total_withdraw,
     )
 
-    with Timeout(seconds=timeout):
+    with transfer_timeout:
```

Comment:
Please, create a new context manager instead of reusing the previous one. Reusing the same instance is probably incorrect